### DISCUSSION TITLE
Help with booting iso file with custom kernel parameter

### DISCUSSION THREAD

**Discussion by Kritiqual** [ventoy/Ventoy#1767] [Q&A] [answered]
I have placed multiple iso file in ventoy (e,g Anarchy.iso and Archlinux.iso, order matter). When I boot into ventoy, select Archlinux iso file, press "e" to edit kernel params, then I press Ctrl + x but the selected iso file is Anarchy iso? How can I boot with custom kernel params and the iso I want, not the first iso?

**Comment by steve6375** [comment, marked as answer]
Use 'grub mode'  Press Ctrl+r  before selecting the ISO.
Then in grub2 menu of the booted ISO you can use 'e', etc.

**Reply by steve6375** [reply]
See also https://ventoy.net/en/doc_name_identifier.html
